db/liquibase/liquibase.properties

```diff
@@ -0,0 +1,6 @@
+driver=org.postgresql.Driver
+classpath=postgresql-42.2.5.jar
+url=jdbc:postgresql://google/avram?ssl=false&socketFactoryArg=broad-avram-dev:us-central1:avram&socketFactory=com.google.cloud.sql.postgres.SocketFactory
```

Comment:
I imagine this will need to be templated somehow to deploy to multiple environments. (Or maybe we have separate ones per env.) Doesn't need to be for this PR, just thinking out loud.

Reply:
We'll also want to be able to run this against a local DB instance.